BondageClub/Scripts/Dialog.js

```diff
@@ -117,32 +117,32 @@ var DialogSelfMenuOptions = [
 /**
  * Compares the player's reputation with a given value
  * @param {string} RepType - The name of the reputation to check
- * @param {string} Value - The value to compare
+ * @param {number} Value - The value to compare
```

Comment:
`CommonDynamicFunctionParams` (see above)
```suggestion
 * @param {string} Value - The value to compare
```